lib/galaxy/datatypes/triples.py

```diff
@@ -202,3 +204,38 @@ def set_peek(self, dataset: DatasetProtocol, **kwd) -> None:
         else:
             dataset.peek = "file does not exist"
             dataset.blurb = "file purged from disk"
+
+
+@build_sniff_from_prefix
+class Sbol(data.Text, Triples):
+    """
+    The SBOL data format
```

Comment:
Adding a link to docs / standard would be appreciated.

Reply:
Agree, added in 6ed37a2c9198256ea558e0c2010e6627c36988de